src/web/backend/rocket_chat/utils.py

```diff
@@ -24,8 +29,32 @@ def rocket(self):
         return get_rocket()
 
 
+def log_rocket_action(event_name):
+    def wrapper(func):
+        @wraps(func)
+        def inner_wrapper(self, **kwargs):
+            res = None
+            success = False
+            message = None
+            try:
+                res = func(self, **kwargs)
+                if res.status_code == 200:
+                    success = True
+            except Exception as e:
+                logger.exception(e)
+                success = False
+                message = str(e)
+            # TODO: what to do with password in create_user method?
+            filtered_kwargs = {key: value for key, value in kwargs.items() if key != 'password'}
```

Comment:
This is a good catch - we may want to store only a partial amount of data, so some input may be required upon retry.